src/tests/ftest/interoperability/diff_versions.py

```diff
@@ -0,0 +1,49 @@
+"""
+  (C) Copyright 2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+from updown_grade_base import UpgradeDowngradeBase
```

Comment:
I'd recommend renaming the base file to `upgrade_downgrade_base` to be inline with `UpgradeDowngradeBase`

Reply:
Done.